pkg/synthetictests/duplicated_events.go

```diff
@@ -64,20 +76,75 @@ var knownEventProblems = []struct {
 	},
 }
 
+type duplicateEventsEvaluator struct {
```

Comment:
if you foresee more uses of the client config here, maybe you can make it part of the struct and not having to plumb it down as an argument

Reply:
> if you foresee more uses of the client config here, maybe you can make it part of the struct and not having to plumb it down as an argument

it's part of the signature on synthetics now, so passing it through is consistent with others.